assets/js/modules/tagmanager/components/common/FormInstructions.js

```diff
@@ -110,37 +98,19 @@ export default function FormInstructions( { showExistingTagMessage } ) {
 		return (
 			<p>
 				{ __(
-					'Looks like your site is using paired AMP. Please select your Tag Manager account and relevant containers below. You can change these later in your settings.',
```

Comment:
@kuasha420 why do we need to remove this text? I don't see it neither in AC nor IB.

Reply:
I didn't realize that AC doesn't cover it but this was an issue spotted by @wpdarren on #5046 where it was saying "You can later change it in the settings" in the Settings page as well. I assumed that the FormInstruction was only used in Settings but It's also used in SEtup. I had made adjustment to it so if it's Setup, only then the `You can change these later in your settings` is shown. Does it look reasonable to you? Cheers.

Reply:
Yes, thanks a lot, @kuasha420.